src/Dapr.AspNetCore/CloudEventsMiddleware.cs

```diff
@@ -138,7 +149,7 @@ private async Task ProcessBodyAsync(HttpContext httpContext, string charSet)
             {
                 httpContext.Request.Body = body;
                 httpContext.Request.ContentType = contentType;
-
+                
```

Comment:
Nit, indentation doesn't need to be here.

Reply:
Fixed